pkgs/top-level/aliases.nix

```diff
@@ -1287,6 +1287,7 @@ mapAliases {
   SPAdes = spades; # Added 2024-06-12
   spark2014 = gnatprove; # Added 2024-02-25
   spatialite_gui = throw "spatialite_gui has been renamed to spatialite-gui"; # Added 2025-01-12
+  spatialite_tools = throw "spatialite_tools has been renamed to spatialite-tools"; # Added 2025-02-06
```

Comment:
It's a general policy to use _ instead of -?

Reply:
This change is aligning "attribute name used for the package" with `pname`. Also, it will be more consistent with [spatialite-gui](https://github.com/NixOS/nixpkgs/blob/master/pkgs/by-name/sp/spatialite-gui/package.nix).

Reply:
fine by me!